.github/workflows/update-homebrew-package.yaml

```diff
@@ -0,0 +1,32 @@
+name: Update Homebrew Package
+
+on:
+  release:
+    types:
+      - released
+
+jobs:
+  update-homebrew-package:
+    name: Update Homebrew Package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Config credentials
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
If we need a token for permissions access, will drop you a line in slack

Reply:
ooooh good call @nrb can help us with this!

Reply:
ooops looks like it is @seemiller that has access

Reply:
I think @seemiller has access to that. I just checked, and I do not. Josh would be a last resort, I know he has admin access.

Reply:
@nrb @dvonthenen Access is granted under the settings tab. I don't have that either. We'll have to find someone else that does.